deployment/ansible/roles/cac-tripplanner.otp-data/tasks/main.yml

```diff
@@ -5,6 +5,11 @@
 - name: Copy Feed Validator
   copy: src=validate_feed.py dest="{{ otp_data_dir }}"
 
+- name: Download OTP Data
+  command: aws s3 sync s3://cleanair-otp-data "{{ otp_data_dir }}/"
+  sudo: no
+  when: production or test
```

Comment:
It would require setting up the AWS account credentials locally (the files aren't public), and might interfere with testing different input files locally.  If neither of those things are a concern, though, then sure, we could run this everywhere.